README.md

```diff
@@ -1,10 +1,9 @@
 # SCISeg: Automatic Segmentation of T2-weighted Intramedullary Lesions in Spinal Cord Injury
 
-[![medRxiv](https://img.shields.io/badge/medRxiv-10.1101/2024.01.03.24300794v2-blue.svg)](https://www.medrxiv.org/content/10.1101/2024.01.03.24300794v2.full.pdf)
+[![medRxiv](https://img.shields.io/badge/medRxiv-10.1101/2024.01.03.24300794v2-blue.svg)](https://www.medrxiv.org/content/10.1101/2024.01.03.24300794v2.full.pdf) [![arXiv](https://img.shields.io/badge/arXiv-2407.17265-b31b1b.svg)](https://doi.org/10.48550/arXiv.2407.17265)
```

Comment:
Or do we want to add `SCIsegV1` and `SCIsegV2` next to the badges?

```suggestion
`SCIsegV1`: [![medRxiv](https://img.shields.io/badge/medRxiv-10.1101/2024.01.03.24300794v2-blue.svg)](https://www.medrxiv.org/content/10.1101/2024.01.03.24300794v2.full.pdf) `SCIsegV2`: [![arXiv](https://img.shields.io/badge/arXiv-2407.17265-b31b1b.svg)](https://doi.org/10.48550/arXiv.2407.17265)
```

Reply:
Yeah I think adding SCIsegV1 and SCIsegV2 next to the badges would be good!

Reply:
Thanks for adding the text.

It seems that markdown formatting does not center the text to the badge:

<img width="300" alt="Screenshot 2024-07-26 at 7 36 00 AM" src="https://github.com/user-attachments/assets/0065deee-8c0c-42ea-8d6a-c5fa6cbbd744">

Maybe we could add a new line?

<img width="300" alt="Screenshot 2024-07-26 at 7 35 51 AM" src="https://github.com/user-attachments/assets/88821332-644c-412a-a3ba-56e460b18cce">

Reply:
https://github.com/ivadomed/model_seg_sci/pull/92/commits/8948b619a9c8359e2f5ef37722e7cccf12c5e3bb found something even better!

Reply:
![Uploading Screenshot 2024-07-26 at 4.02.49 PM.png…]()

Reply:
<img width="836" alt="Screenshot 2024-07-26 at 4 03 23 PM" src="https://github.com/user-attachments/assets/6e6432e6-0e2d-4654-8d53-94b3f8c44758">

Reply:
nice! merging!